### PR TITLE
 feature/BNGP-4219-suppress-metric-validation

### PR DESCRIPTION
**Pull request by soorajvdefra**

### CODE REVIEW

File: packages/webapp/src/utils/helpers.js [hunk 1]
```diff
@@ -613,7 +613,10 @@ const getMetricFileValidationErrors = (metricValidation, href, useStatutoryMetri
   } else if (!metricValidation.isOffsiteDataPresent) {
     error.err[0].text = 'The selected file does not have enough data'
   } else if (!metricValidation.areOffsiteTotalsCorrect) {
-    error.err[0].text = 'The selected file has an error - the baseline total area does not match the created and enhanced total area for the off-site'
+    console.log('sooraj')
```
Comment (paulsimonandrews) [inline packages/webapp/src/utils/helpers.js:616]: Remove the console log

**Review by paulsimonandrews — COMMENTED** [review]
Tiny change to remove console log.

Code looks fine, before approving/merging would like to check with Chris that the validation will be re-introduced. If definitely not, we should remove it properly, not just disable as the code does here.

File: packages/webapp/src/utils/helpers.js (hunk 1 above)
Comment (soorajvdefra) [reply packages/webapp/src/utils/helpers.js]: console.log removed

**Review by paulsimonandrews — APPROVED** [review]
LGTM

**Comment by shaundefra** [comment]
tested locally

**Pull request merged**